specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2024-10-01-preview/examples/ApiManagementListGatewaysBySubscriptionAndResourceGroup.json

```diff
@@ -69,13 +69,44 @@
                 }
               },
               "configurationApi": {
-                "hostname": "standard-gw-2.eastus.configuration.gateway.azure-api.net"
+                "hostname": "standard-gw-2.configuration.eastus-01.azure-api.net"
               }
             },
             "sku": {
               "name": "Standard",
               "capacity": 1
             }
+          },
+          {
+            "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.ApiManagement/gateways/standard-gw-2",
+            "name": "standard-gw-2",
+            "type": "Microsoft.ApiManagement/gateways",
+            "tags": {
+              "Owner": "vitaliik"
+            },
+            "location": "East US",
+            "etag": "AAAAAAAWKwo=",
+            "properties": {
+              "provisioningState": "Succeeded",
+              "targetProvisioningState": "",
+              "createdAtUtc": "2021-06-16T09:40:00.9453556Z",
+              "frontend": {
+                "defaultHostname": "standard-gw-2.eastus.gateway.azure-api.net"
+              },
+              "backend": {
+                "subnet": {
+                  "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.Network/virtualNetworks/vn2/subnets/sn2"
+                }
+              },
+              "configurationApi": {
+                "hostname": "standard-gw-2.configuration.eastus-01.azure-api.net"
+              }
```

Comment:
```suggestion
              }
```

Reply:
Done, though I left the other two untouched as I believe they represent something else?